Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -6,6 +6,8 @@
 import requests
 import json
 
+handle_proxy()
```

Comment:
better to keep it in main.
initialize the `SESSION` global within the main as well.

Reply:
after this fix, let's do a quick demo